profile/gperf.go

```diff
@@ -0,0 +1,132 @@
+package profile
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"strings"
+	"time"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// Common flags of perf
+const (
```

Comment:
I don't understand why we need them here.

Reply:
Removed